user-manual/add-edit-content/authority-records.rst

```diff
@@ -579,38 +580,38 @@ You will be redirected to the :term:`view page` for the edited
 Add access points to your authority record
 ==========================================
 
-The International Council on Archives' :ref:`ISAD(G) <isad-template>` standard 
-defines an :term:`access point` as a "*name, term, keyword, phrase or code that 
-may be used to search, identify and locate an archival description.*" Access 
+The International Council on Archives' :ref:`ISAD(G) <isad-template>` standard
+defines an :term:`access point` as a "*name, term, keyword, phrase or code that
+may be used to search, identify and locate an archival description.*" Access
 points help users not only understand contextual information, but discover further
-related content. In AtoM, you can now apply access points to an 
-:term:`authority record` as well. 
+related content. In AtoM, you can now apply access points to an
+:term:`authority record` as well.
 
-In AtoM, access points are controlled value :term:`terms <term>` maintained in a 
+In AtoM, access points are controlled value :term:`terms <term>` maintained in a
 :term:`taxonomy`, which can be used to aid in discovery. Currently AtoM supports
-one type of access point for authority records - Occupations. By default in a 
```

Comment:
uhhhhhh uh oh - even though it was merged earlier, I don't see your previous work on updating the access points in here!